hs_docker_base

```diff
@@ -1 +1 @@
-Subproject commit 0b178b09012f38773a300bc290d4894074d4b3bc
+Subproject commit 39bfe8c29860e008fb8e975eb5d2b1b4ee88ff57
```

Comment:
it is always unintended and I'm never sure how to correct it. I pulled the latest changes in hs_docker_base and committed, but I'll try again. Please advise if the correction is something else.